### PR TITLE
Added meeting duration timer

### PR DESCRIPTION
**Pull request by Dev-Mikko**
Added a meeting duration timer in the room. The needed room-server function are implemented by [PR #4](https://github.com/edumeet/edumeet-room-server/pull/4). The result is the following:

**Desktop**

immagine

**Mobile**

immagine

### CODE REVIEW

**Comment by havfo** [comment]
How does this handle if the room-server is in a different time-zone than the client?

**Comment by Dev-Mikko** [comment]
The timestamps are in UTC datetime format, since I'm computing them using Date.now() method (see [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now)). I think that there is not any problem with geographic distribution.

**Comment by Dev-Mikko** [comment]
OK, I do not considered this aspect. I'll fix it ASAP

**Comment by Dev-Mikko** [comment]
@havfo I think that there is no problem. I'm using only the timestamps without converting them into local dates. All the timestamps are timezone-independent. So the operations are performed correctly even if the server is in a different timezone with respect to the clients.

File: src/components/topbar/TopBar.tsx [hunk 1]
```diff
@@ -113,6 +122,51 @@ const TopBar = ({
...
+				setMeetingDuration(Date.now() - roomCreationTimestamp);
+
+				setTimeout(driftAwareTimer, Math.max(0, interval - dt));
+			};
+
+			const computeDuration = setTimeout(driftAwareTimer, interval);
+
+			return () => clearInterval(computeDuration);
+		}
+	});
```
Comment (havfo) [inline src/components/topbar/TopBar.tsx:168]: You need to add an empty dependency array here `, []` or this will execute after every re-render.

Comment (Dev-Mikko) [reply src/components/topbar/TopBar.tsx]: Done

**Review by havfo — APPROVED** [review]
LGTM

**Pull request merged**